### PR TITLE
Allow PHP 8

### PR DESCRIPTION
**Pull request by mnapoli**

### CODE REVIEW

**Comment by Kleinast** [comment]
It's seem FriendsOfPHP/PHP-CS-Fixer doesn't support php 8 yet (https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4702)
But if it's composer requirement problem you have like here https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/5326, I can accept if you change only the composer.json in your PR.

File: .travis.yml [hunk 1]
```diff
@@ -3,6 +3,7 @@ language: php
 php:
   - 7.3
   - 7.4
+  - 8.0
```
Comment (Kleinast) [inline .travis.yml:6]:
```suggestion
```

**Review by Kleinast — CHANGES_REQUESTED** [review]

**Comment by mnapoli** [comment]
Make sense, I updated it thanks!

**Review by Kleinast — APPROVED** [review]

**Comment by mnapoli** [comment]
Thanks!

**Comment by Kleinast** [comment]
My pleasure, thanks for the contribution.

**Pull request merged**